package.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>ament_cmake_doxygen</name>
+  <version>0.6.2</version>
+  <description>CMake macros to help generating documentation with doxygen</description>
+  <maintainer email="daniel.stonier@tri.global">Daniel Stonier</maintainer>
+  <license>TODO</license>
```

Comment:
BSD?

Reply:
I think this is mostly to you, @stonier .

Reply:
Apache 2.0 to match ament_cmake's license works for me.